client/src/assets/styles/global.scss

```diff
@@ -1,9 +1,19 @@
+/** Colors */
+/** TODO: Move colors to copper.*/
+$white: #FFFFFF;
+
+$light-blue-800: #0576BD;
+
 /** Typography */
 
 .h1-header {
   font-size: 24px;
 }
 
+.h1-header-xxl {
+  font-size: 80px;
```

Comment:
This is reall big - even bigger than the logo I think. I think it's fine to keep for now I'm just adding a task to lead out to standardize these fonts. Can you do the same thing you did elsewhere 4px * xxx?

Reply:
Made 56px instead (what I previously added as h1-header-xl). Will do a followup PR to remove from landing page styles I submitted but yes will fix here. Thanks!